LibreNMS/Snmptrap/Handlers/LinkUp.php

```diff
@@ -53,7 +53,7 @@ public function handle(Device $device, Trap $trap)
             return;
         }
 
-        $port->ifOperStatus = $trap->getOidData("IF-MIB::ifOperStatus.$ifIndex") ?: 'up';
```

Comment:
From reading the issue it looks like up should still use what's in the trap, it's just the down one that needs to change.

Reply:
Indeed ! I should stop trying to use my brain after midnight for a while , I guess :D 
Will try to fix that now (breaking the above statement already :) )